docs/devices-api/getting-started/apple-health.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "Apple Health"
+description: "Get Apple Health data via webhooks."
+---
+
+## Overview
+
+This is a guide to help you integrate with Apple Health - a repository for health and fitness data available on iPhone and iPad devices.
+We provide an SDK that allows you to receive Apple Health data in the background via webhooks. When a user adds health data,
+the SDK will automatically send it to the Metriport API, and we'll forward it to your webhook URL.
+
```

Comment:
Should we make any mention maybe via a tip or info block of how the user needs to go back into the app for data to get sent?